fission-web-server/library/Fission/Web/Server.hs

```diff
@@ -57,10 +56,6 @@ import           Fission.Internal.Orphanage.PlainText   ()
 type API    = Swagger.API :<|> Fission.API :<|> LinkWS
 type LinkWS = "user" :> "link" :> RelayWS
 
--- | Run actions described by a @Fission@ type
-runServer :: MonadIO m => Config -> Fission.Server a -> m a
-runServer cfg actions = runRIO cfg $ unServer actions
-
```

Comment:
Moved into the `Fisssion.Web.Server.Types` module, since we need it in there sometimes